server/player/player.go

```diff
@@ -964,6 +989,26 @@ func (p *Player) Respawn() {
 	p.SetVisible()
 }
 
+// SpawnPos returns spawn position in the current world.
+func (p *Player) SpawnPos() cube.Pos {
+	return p.World().PlayerSpawn(p.uuid)
+}
+
+// SetSpawnPos sets spawn position for the player in the provided world.
+func (p *Player) SetSpawnPos(pos cube.Pos, w *world.World) {
+	w.SetPlayerSpawn(p.UUID(), pos)
+}
```

Comment:
Does it make sense for SpawnPos() to return the current world and then SetSpawnPos() take in any world? Might be worth making SpawnPos() also take in a world just to be consistent